Makefile

```diff
@@ -0,0 +1,32 @@
+.PHONY: build coverage lint graphs tests docs clean clean_cython deploy
+
+build:
+	mkdir -p docs/modules/generated
+	python setup.py build_ext --inplace
+
+coverage:
+	nosetests --logging-level=INFO --with-coverage --cover-package=discretize --cover-html
+	open cover/index.html
+
+lint:
+	pylint --output-format=html discretize > pylint.html
+
+graphs:
+	pyreverse -my -A -o pdf -p discretize discretize/**.py discretize/**/**.py
+
+tests:
+	nosetests --logging-level=INFO
+
+docs:
+	cd docs;make html
+
+clean:
+	cd docs;make clean
+	find . -name "*.pyc" | xargs -I {} rm -v "{}"
+
+clean_cython: clean
+	find . -name "*.c" | xargs -I {} rm -v "{}"
+	find . -name "*.so" | xargs -I {} rm -v "{}"
+
+deploy:
+	python setup.py sdist bdist_wheel upload
```

Comment:
do we want to use `twine` ?
https://packaging.python.org/distributing/

![image](https://cloud.githubusercontent.com/assets/6361812/21776507/cb3fc0c6-d64f-11e6-8283-ede18e0ae3dd.png)